src/components/DebtPool/DebtPool.tsx

```diff
@@ -43,25 +55,15 @@ export const DebtPool: React.FC<IProps> = ({ title, subTitle, data }) => {
                 arcLabelsRadiusOffset={0}
                 colors={{ datum: 'data.color' }}
                 arcLabelsTextColor='#000000'
-                tooltip={({ datum: { color, label } }) => (
+                tooltip={() => (
```

Comment:
I think, null as value of this prop should be enough

Reply:
tooltip prop must returned RC, so i add a disabled div. If i add null as value TS will throw an error.

Reply:
maybe just function returning null? something like that is used in collateral plot, check it